src/main/java/it/pagopa/pn/deliverypush/middleware/externalclient/pnclient/externalchannel/ExternalChannelSendClientImpl.java

```diff
@@ -147,7 +148,7 @@ private void sendNotificationPEC(String requestId, NotificationInt notificationI
 
             log.info("[exit] sendNotificationPEC address={} requestId={} recipient={}", LogUtils.maskEmailAddress(digitalAddress.getAddress()), requestId, LogUtils.maskGeneric(recipientInt.getDenomination()));
         } catch (Exception e) {
-            throw new PnInternalException("error sending PEC notification", e);
+            throw new PnInternalException("error sending PEC notification", ERROR_CODE_DELIVERYPUSH_SENDPECNOTIFICATIONFAILED);
```

Comment:
- Nei casi in cui l'exception è presente va sempre utilizzato il costruttore di PnInternalException che prende anche il throwable come parametro di ingresso, altrimenti ci si perde l'informazione sull'exception.
- Aggiungere Log errore con exception